ci/tizen/generate_sysroot.py

```diff
@@ -18,6 +18,8 @@
     'glibc',
     'glibc-devel',
     'libgcc',
+    'libstdc++',
+    'libstdc++-devel',
```

Comment:
The devel package should not be necessary.

Reply:
I removed devel package